### PR TITLE
Update Monaco editor colors

### PR DESCRIPTION
**Pull request by spadgett**
https://jira.coreos.com/browse/CONSOLE-1621

/assign @rhamilto 
cc @bmignano 

Screen Shot 2019-07-26 at 4 44 05 PM

### CODE REVIEW

**Comment by bmignano** [comment]
This is looking great! I'll update here when we have a new color for boolean values from Mary. I'm undecided on the white border vs. no border. It actually looks okay to me without it, but adding it would make the page more in line with the Logs and Terminal tabs. Thoughts @openshift/team-ux-review ?

**Comment by spadgett** [comment]
I wonder if we should remove the white margin around logs and terminal so there's more room for content. We take away so much space with the masthead/nav/notifications/titles/tabs I'm hesitant to add additional margins. It's really important for editors.

**Comment by spadgett** [comment]
We should look at the context menu styles as well, although I'm having trouble customizing them:

Screen Shot 2019-07-26 at 5 04 01 PM

**Comment by spadgett** [comment]
Added `#CBC0FF` for keywords (boolean values)

etcd-operator-5c4b6b47c9-v7hcq · Details · OKD 2019-07-26 17-42-33

**Comment by serenamarie125** [comment]
Color updates look great! @beanh66 talked to me about adding a margin so it's more similar to logs and terminals, and i think that would be a great suggestion.  This would help the issue with the nav/editor touching.

What does this look like in read only mode?

File: frontend/public/components/edit-yaml.jsx [hunk 1]
```diff
@@ -7,6 +7,11 @@ import { connect } from 'react-redux';
 import MonacoEditor from 'react-monaco-editor';
 import { ActionGroup, Alert, Button } from '@patternfly/react-core';
 import { DownloadIcon } from '@patternfly/react-icons';
+import {
+  global_BackgroundColor_100 as lineNumberActiveForeground,
+  global_BackgroundColor_300 as lineNumberForeground,
+  global_BackgroundColor_dark_100 as editorBackground,
```
Comment (rhamilto) [inline frontend/public/components/edit-yaml.jsx]: Looks like this import is unused.

Comment (spadgett) [reply frontend/public/components/edit-yaml.jsx]: @rhamilto it's used (line 39)

**Comment by bmignano** [comment]
@spadgett agreed! Is it possible to change the color of the text in the context menu? Mary suggests either white or #ededed to increase contrast.

**Pull request merged**